travvy-server/src/models/index.js

```diff
@@ -20,7 +23,14 @@ fs
     const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes)
     db[model.name] = model
   })
-
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.')
```

Comment:
ok yeah this is good for feedback on the console. Just to make sure we have to npm run serve both in travvy and in travvy-server each time?

Reply:
it would be "npm start" for the backend if im not mistaken, but 'npm run serve' for the Vue application

Reply:
This message is with respect to the database connection being successful, however there is another one for the starting of the app